doc/conf.py

```diff
@@ -8,6 +8,8 @@
 
 
 _g = globals()
-_g.update(build_package_configs(
-    project_name='ts_MTAlignment',
-    version=lsst.ts.MTAlignment.version.__version__))
+_g.update(
+    build_package_configs(
+        project_name="ts_MTAlignment", version=lsst.ts.MTAlignment.version.__version__
+    )
+)
```

Comment:
I think this is outdated code and should not be used like this anymore. However, since the project documentation is not created yet (there is no Jenkinsfile) let's leave it like this.